Formula/mirrord.rb

```diff
@@ -0,0 +1,20 @@
+class Mirrord < Formula
+  desc "Connect your local process and your cloud environment"
```

Comment:
```suggestion
  desc "Connect your local process and your cloud environment, and run local code in cloud conditions"
```

Reply:
we cant do this, the description has a limit
`brew audit --new --online mirrord`